libs/sdk/skyux-eslint/package.json

```diff
@@ -25,7 +25,19 @@
     "@typescript-eslint/utils": "^8.30.1"
   },
   "dependencies": {
-    "@skyux/manifest": "0.0.0-PLACEHOLDER",
+    "@skyux-sdk/eslint-schematics": "0.0.0-PLACEHOLDER",
     "tslib": "^2.8.1"
+  },
+  "ng-add": {
+    "save": "devDependencies"
+  },
+  "schematics": "@skyux-sdk/eslint-schematics/collection.json",
```

Comment:
Created a separate "schematics" library and point to those schematics in `skyux-eslint`. This was done so that Angular CLI consumers can run `ng add skyux-eslint`, but non-Angular consumers won't have to install Angular CLI dependencies to use `skyux-eslint` as-is.